locales/es/messages.po

```diff
@@ -3227,15 +3227,31 @@ msgstr "<0>Si tu apartamento es actualmente de renta estabilizada, o lo ha sido
 msgid "justfix.whyIsPhoneNumberNeeded"
 msgstr "Utilizaremos esta información para:<0><1>Acceder a tu cuenta actual</1><2>Comprobar si coincide con una cuenta preexistente </2><3>Abrirte una cuenta nueva.</3></0><4/>Una cuenta te permitirá volver a nuestras herramientas en cualquier punto del proceso sin tener que empezar desde el principio, descargar cualquier documentos que hayas completado, y recibir notificaciones sobre cambios relevantes a políticas que te afecten."
 
-#: frontend/lib/laletterbuilder/letter-builder/habitability/habitability-letter-content.tsx:16
+#: frontend/lib/laletterbuilder/letter-builder/habitability/habitability-letter-content.tsx:17
 msgid "laletterbuilder.emailToLandlordBody"
 msgstr "<0>Por favor vea la carta adjunta de <1/>. </0><2>Para documentar las comunicaciones y evitar malentendidos, por favor corresponda con <3/> por correo electrónico a <4>{0}</4> o por correo ordinario en lugar de una llamada telefónica o una visita en persona. </2>"
 
 #: frontend/lib/laletterbuilder/about.tsx:54
 msgid "laletterbuilder.explanationAboutWhyWeMadeThisSite"
 msgstr "We made this site because xyz"
 
-#: frontend/lib/laletterbuilder/letter-builder/habitability/habitability-letter-content.tsx:80
+#: frontend/lib/laletterbuilder/letter-builder/habitability/habitability-letter-content.tsx:95
+msgid "laletterbuilder.habitability.access-intro"
+msgstr ""
+
+#: frontend/lib/laletterbuilder/letter-builder/habitability/habitability-letter-content.tsx:90
+msgid "laletterbuilder.habitability.access-title"
+msgstr ""
+
+#: frontend/lib/laletterbuilder/letter-builder/habitability/habitability-letter-content.tsx:102
+msgid "laletterbuilder.habitability.access-warning"
+msgstr ""
+
+#: frontend/lib/laletterbuilder/letter-builder/habitability/habitability-letter-content.tsx:115
+msgid "laletterbuilder.habitability.consequences"
+msgstr ""
+
+#: frontend/lib/laletterbuilder/letter-builder/habitability/habitability-letter-content.tsx:81
 msgid "laletterbuilder.habitability.intro"
 msgstr "LETTER TEXT"
```

Comment:
@samaratrilling I was seeing that this was not deleted?? maybe im not catching something?